outputs/redis.go

```diff
@@ -0,0 +1,63 @@
+package outputs
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/falcosecurity/falcosidekick/types"
+	"github.com/redis/go-redis/v9"
+)
+
+func NewRedisClient(config *types.Configuration, stats *types.Statistics, promStats *types.PromStatistics,
+	statsdClient, dogstatsdClient *statsd.Client) (*Client, error) {
+
+	ctx := context.Background()
+
+	rClient := redis.NewClient(&redis.Options{
+		Addr:     config.Redis.Address + ":" + config.Redis.Port,
+		Password: config.Redis.Password,
```

Comment:
Is it working even if `config.Redis.Password` is empty?

Reply:
Yes @Issif, It does.